client/payment-details/summary/index.tsx

```diff
@@ -460,6 +474,41 @@ const PaymentDetailsSummary: React.FC< PaymentDetailsSummaryProps > = ( {
 								{ charge.payment_intent
 									? charge.payment_intent
 									: charge.id }
+								{ ! charge?.refunded && (
+									<DropdownMenu
+										icon={ moreVertical }
+										label="more"
+										popoverProps={ {
+											position: 'bottom left',
+										} }
+									>
+										{ () => (
+											<MenuGroup>
+												<MenuItem
+													onClick={ () =>
+														setIsRefundModalOpen(
+															true
+														)
+													}
+												>
+													Refund in full
+												</MenuItem>
+												{ charge.order && (
+													<MenuItem
+														onClick={ () =>
+															( window.location =
+																// eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
What TS was complaining about here?

Reply:
That `charge.order` can possibly be null but it's it's in practise impossible since we have condition for that.